admin/tools/docker/base/Dockerfile

```diff
@@ -202,23 +200,22 @@ RUN dnf install -y 'dnf-command(config-manager)' \
         openssl \
         procps-ng \
         protobuf \
-        python38 \
+        python3 \
+        pip \
         tree \
         vim \
-    && dnf module install -y mariadb/client \
+    && dnf install -y mariadb \
```

Comment:
Please check that this is client-only, to make sure we don't bloat the containers?

Reply:
Tou're right - `mariadb` would install the server as well. It looks like this is a super-package that includes `mariadb-server` and client tools. I will further investigate this. In theory, a special client-only package `mysql` could be used. 

I just couldn't get this to work in AlmaLinux 9:
```
dnf module install -y mariadb/client
```

Reply:
For the record. Replaced `mariadb` with the client package `mysql`. Verified that Qserv works after the change.